python/lsst/sims/maf/db/opsimDatabase.py

```diff
@@ -3,6 +3,7 @@
 import warnings
 from .Database import Database
 from lsst.sims.maf.utils import getDateVersion, TelescopeInfo
+from sqlalchemy import text
```

Comment:
Good :) I guess you can also check
isinstance(constraint, text) to see if you should not try to convert it.

On Fri, Aug 28, 2015 at 4:12 PM Peter Yoachim <notifications@github.com>
wrote:

> In python/lsst/sims/maf/db/opsimDatabase.py
> <https://github.com/lsst/sims_maf/pull/33#discussion_r38253221>:
>
> > @@ -3,6 +3,7 @@
> >  import warnings
> >  from .Database import Database
> >  from lsst.sims.maf.utils import getDateVersion, TelescopeInfo
> > +from sqlalchemy import text
>
> yup, that's much better. I got confused by error messages and thought it
> didn't work there, but it turns out sqlalchemy just doesn't let you convert
> a text to a text.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/lsst/sims_maf/pull/33/files#r38253221>.
>